warehouse/accounts/views.py

```diff
@@ -53,6 +53,7 @@ def failed_logins(exc, request):
 @view_config(
     route_name="accounts.profile",
     renderer="accounts/profile.html",
+    uses_session=True,
```

Comment:
@dstufft mentioned [here](https://github.com/pypa/warehouse/pull/1444#r76335326) that he'd prefer not to turn on sessions for this page, however I'm not sure what the alternative would be.